frontend/src/components/interaction-manager/content/log.tsx

```diff
@@ -6,13 +6,13 @@ import type { InteractionContentProps, Log } from '../types'
 
 const getMessageIntent = (type: string) => {
```

Comment:
suggestion (non blocking):

```suggestion
const getMessageIntent = (type: LogContent['type']) => {
```

Reply:
makes sense, thanks!